src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {
+    Green,
+    Red,
+    Blue,
+    Black,
+    Yellow,
+    White,
+    Orange,
+    Pink
+}
```

Comment:
```suggestion
    GREEN,
    RED,
    BLUE,
    BLACK,
    YELLOW,
    WHITE,
    ORANGE,
    PINK
}
```
enums should have names in uppercase